src/legacy/core_plugins/console/public/np_ready/lib/utils/utils.ts

```diff
@@ -81,39 +79,39 @@ utils.collapseLiteralStrings = function (data) {
 
 const LITERAL_STRING_CANDIDATES = /((:[\s\r\n]*)([^\\])"(\\"|[^"\n])*\\?")/g;
 
-utils.expandLiteralStrings = function (data) {
-  return data.replace(LITERAL_STRING_CANDIDATES, function (match, string) {
+export function expandLiteralStrings(data: string) {
+  return data.replace(LITERAL_STRING_CANDIDATES, (match, string) => {
     // Expand to triple quotes if there are _any_ slashes
     if (string.match(/\\./)) {
       const firstDoubleQuoteIdx = string.indexOf('"');
       const colonAndAnySpacing = string.slice(0, firstDoubleQuoteIdx);
       const rawStringifiedValue = string.slice(firstDoubleQuoteIdx, string.length);
       const jsonValue = JSON.parse(rawStringifiedValue)
-        .replace('^\s*\n', '')
-        .replace('\n\s*^', '');
+        .replace('^s*\n', '')
+        .replace('\ns*^', '');
```

Comment:
I don't understand why the backslashes have been removed here, it looks like a mistake? And in the second replace, it looks like the caret (`^`) ought to be a dollar (`$`).

Then again, these two `.replace` lines can be replaced with a simple `.trim()`.

Reply:
So this is an instance where prettier snuck in a change 🤕 great catch!

The behaviour described by these replace functions is not the same is `trim` with respect to new lines but you are correct regarding the caret symbol! Will update.